recipe/meta.yaml

```
@@ -32,10 +29,30 @@ requirements:
     - python
 
 test:
+  # this section automatically executes run_test.py (from recipe)
+  files:
+    - fib.pyx      # needed by run_test.py
+  source_files:
+    - runtests.py  # from repo
+    # files below needed by runtests.py
+    - cython.py
+    - tests/
+    - Cython/Runtime/refnanny.pyx
+    - Demos/embed
+  requires:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - setuptools
+    - numpy
+    - pythran
+    - tbb-devel  # [linux]
```

Might be worth trying this to see if tests pass with older TBB versions that are compatible with GCC 9.4.0:
```suggestion
    - tbb-devel<2021  # [linux]
```